script.js

```diff
@@ -1,4 +1,62 @@
+var sciSSorsPaperStone = function () {
+  var randomNumber = Math.floor(Math.random() * 3 + 1);
```

Comment:
Nice that you combined Math.floor() and Math.random() in a single expression to generate _randomNumber_.